### PR TITLE
Add mention of mmap perf change

### PR DESCRIPTION
**Pull request by zkat**

### CODE REVIEW

**Comment by anangaur** [comment]
Docs Build status updates of commit _[cd6e6ef](https://github.com/NuGet/docs.microsoft.com-nuget/commits/cd6e6ef28e28f13522174b5c9b53af0576c18e0d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-5.8.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-zkat-mention-mmap/docs/release-notes/NuGet-5.8.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.8?branch=pr-en-us-2211) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C12%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202011121911413583-2211%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2Fu%2FvXg47bAvZmI3H5u%2B3v3hhmRMcgCYtphtSG%2FC7e%2Fw%3D&st=2020-11-12T19%3A07%3A15Z&se=2020-12-13T19%3A12%3A15Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/release-notes/NuGet-5.8.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ NuGet distribution vehicles:
 
 * Several other NuGet performance improvements in Visual Studio - [#9982](https://github.com/NuGet/Home/issues/9982), [#9984](https://github.com/NuGet/Home/issues/9984), [#10052](https://github.com/NuGet/Home/issues/10052), [#9903](https://github.com/NuGet/Home/issues/9903)
 
+* Non-No-Op restore performance was improved drastically (on the order of 50% faster) by using `MemoryMappedFile`s during extraction - [#9807](https://github.com/NuGet/Home/issues/9807)
```
Comment (aortiz-msft) [inline docs/release-notes/NuGet-5.8.md:43]: Regular NuGet restore performance?
```suggestion
* Regular NuGet restore performance was improved up significantly (32% for OrchardCore; 40% for our own Nuget.Client repo) by using `MemoryMappedFile`s during extraction - [#9807](https://github.com/NuGet/Home/issues/9807)
```

**Review by aortiz-msft — CHANGES_REQUESTED** [review]

**Comment by anangaur** [comment]
Docs Build status updates of commit _[c170bcc](https://github.com/NuGet/docs.microsoft.com-nuget/commits/c170bcc58198e2934fcaab57865fd8e8435e644b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-5.8.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-zkat-mention-mmap/docs/release-notes/NuGet-5.8.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.8?branch=pr-en-us-2211) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C12%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202011121932511158-2211%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=z1T5ruUW9I%2B6JFHPxZ1DR6bURFs2nAYKrQZadkNdBOE%3D&st=2020-11-12T19%3A28%3A22Z&se=2020-12-13T19%3A33%3A22Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by aortiz-msft — APPROVED** [review]
@JonDouglas - Any other feedback?

**Comment by JonDouglas** [comment]
We created PRs at the same time. I merged https://github.com/NuGet/docs.microsoft.com-nuget/pull/2212 already!

**Comment by JonDouglas** [comment]
Going to close as duplicate is already live.

**Pull request closed without merging**